app/graph/mutations/project_media_mutations.rb

```diff
@@ -18,12 +18,12 @@ module ProjectMediaMutations
     project_id: 'int',
     previous_project_id: 'int',
     copy_to_project_id: 'int',
-    add_to_project_id: 'int',
+    move_to_project_id: 'int',
     remove_from_project_id: 'int',
     refresh_media: 'int',
     archived: 'int',
     metadata: 'str'
   })
 
-  Create, Update, Destroy = GraphqlCrudOperations.define_crud_operations('project_media', create_fields, update_fields, ['project', 'project_was', 'check_search_team', 'check_search_trash', 'check_search_project', 'check_search_project_was', 'relationships_target', 'relationships_source', 'related_to'])
+  Create, Update, Destroy = GraphqlCrudOperations.define_crud_operations('project_media', create_fields, update_fields, ['project_was', 'check_search_team', 'check_search_trash', 'check_search_project', 'check_search_project_was', 'relationships_target', 'relationships_source', 'related_to'])
```

Comment:
what about `project_was` ? @caiosba

Reply:
This project set when move from one project to another one so I kept it

Reply:
ok, so why not keep `project` too in this case?

Reply:
I expected to know the current projects via `projects` list but `project_was` was set when calling move to another project
Let's know what @caiosba think?